dd-sdk-android/src/main/kotlin/com/datadog/android/log/internal/logger/DatadogLogHandler.kt

```diff
@@ -22,7 +24,8 @@ internal class DatadogLogHandler(
     internal val networkInfoProvider: NetworkInfoProvider?,
     internal val timeProvider: TimeProvider,
     internal val userInfoProvider: UserInfoProvider,
-    internal val bundleWithTraces: Boolean = true
+    internal val bundleWithTraces: Boolean = true,
+    internal val bundleWithRum: Boolean = true
```

Comment:
Do we need those defaults? Now defaults come from two places (`Logger.Builder` and here from the initializer).

Reply:
Better safe than sorry. ;)